src/wallet/wallet.cpp

```diff
@@ -1921,8 +1921,8 @@ BlindingStatus CWallet::WalletBlindPSBT(PartiallySignedTransaction& psbtx) const
             our_input_data[i] = std::make_tuple(amount, asset, asset_blinder, value_blinder);
         }
 
-        // Blind issuances on our inputs
-        if (input.m_issuance_value || input.m_issuance_inflation_keys_amount) {
```

Comment:
Leaving this logic in place even though blinding is removed, so that when the PSET spec is updated we can easily add blinding back.